egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java

```diff
@@ -525,6 +525,10 @@ public String loadvoucherview() throws ApplicationException {
 
        //heading = ReportUtil.getCityName();
        heading = microserviceUtils.getHeaderNameForTenant().toUpperCase();
+       final EgBillregister billRegister = (EgBillregister) persistenceService
+               .find("from EgBillregister br where br.egBillregistermis.voucherHeader.id=" + voucherHeader.getId());
```

Comment:
Please don't append the value to the query directly. Use named parameter or positional parameter.

Reply:
changes committed.Please review